code/ObjFileImporter.cpp

```diff
@@ -414,9 +414,13 @@ void ObjFileImporter::createVertexArray(const ObjFile::Model* pModel,
     // Copy vertices of this mesh instance
     pMesh->mNumVertices = numIndices;
     if (pMesh->mNumVertices == 0) {
+        delete pMesh;
+        pMesh = nullptr;
         throw DeadlyImportError( "OBJ: no vertices" );
-    } else if (pMesh->mNumVertices > AI_MAX_ALLOC(aiVector3D)) {
```

Comment:
AI_MAX_ALLOC exists because Valgrind and AddressSanitizer have a maximum allocation size and exceeding it makes `new` return `nullptr` which breaks the normal rules. Changing this check breaks running under asan/valgrind. It's not possible to check every return value of `new` against `nullptr` and it would cause warnings from other tools since it can't happen in normal operation. However the value of AI_MAX_ALLOC could be increased or even moved to config.h.

Reply:
ok, didn't know about this. Will revert this change.